src/Entity/IdeasWorkshop/Idea.php

```diff
@@ -29,7 +29,51 @@
 /**
  * @ApiResource(
  *     collectionOperations={
- *         "get": {"method": "GET"},
+ *         "get": {
+ *             "method": "GET",
+ *             "requirements": {"id": "%pattern_uuid%"},
+ *             "swagger_context": {
+ *                 "parameters": {
+ *                     {
+ *                         "name": "status",
+ *                         "in": "query",
+ *                         "type": "string",
+ *                         "description": "The status of the Idea resource.",
+ *                         "enum": IdeaStatusEnum::ALL_STATUSES,
+ *                         "example": IdeaStatusEnum::PENDING,
+ *                     },
+ *                     {
+ *                         "name": "name",
+ *                         "in": "query",
+ *                         "type": "string",
+ *                         "description": "The name of the Idea resource.",
+ *                         "example": "écologie",
+ *                     },
+ *                     {
+ *                         "name": "theme.name",
+ *                         "in": "query",
+ *                         "type": "string",
+ *                         "description": "The theme name of the Idea resource.",
+ *                         "example": "armée",
+ *                     },
+ *                     {
+ *                         "name": "author_category",
+ *                         "in": "query",
+ *                         "type": "uuid",
+ *                         "description": "The author category of the Idea resource.",
+ *                         "enum": AuthorCategoryEnum::ALL_CATEGORIES,
```

Comment:
No need to create another constant, you can do `AuthorCategoryEnum::toArray()` to get all categories.

Reply:
i tried this before, but callbacks are not handled here. only a constant would do the trick